Src/Json/Shared.Json/ObjectDiffPatch.cs

```diff
@@ -0,0 +1,378 @@
+﻿#region *   License     *
+/*
+    SimpleHelpers - ObjectDiffPatch   
```

Comment:
You mean this https://github.com/dennisdoomen/fluentassertions/wiki#object-graph-comparison ? Yes, that would be the best place to fit in. 

And you're right: it boils down to comparing property graphs just as dictionaries. So, there is a good chance we can use some of FA's built-in stuff and leave out the package. I will look into this.